<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="CL-NB-353" name="belcorp.test.CarritoBolsaCompra.CPA_18_Eliminar_Prod_Pagar_CheckOut" tests="1" failures="0" timestamp="12 Oct 2017 18:43:25 GMT" time="5.011" errors="1">
  <testcase name="CPA_18" time="5.011" classname="belcorp.test.CarritoBolsaCompra.CPA_18_Eliminar_Prod_Pagar_CheckOut">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.CPA_18() didn&#039;t finish within the time-out 5000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.CPA_18() didn't finish within the time-out 5000
	at java.io.WinNTFileSystem.createFileExclusively(Native Method)
	at java.io.File.createTempFile(Unknown Source)
	at org.openqa.selenium.io.TemporaryFilesystem.createTempDir(TemporaryFilesystem.java:83)
	at org.openqa.selenium.firefox.FirefoxProfile.layoutOnDisk(FirefoxProfile.java:417)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:85)
	at org.openqa.selenium.firefox.FirefoxDriver.startClient(FirefoxDriver.java:271)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:119)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:216)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:211)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:207)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at belcorp.utils.BrowserFactory.startBrowser(BrowserFactory.java:25)
	at belcorp.test.CarritoBolsaCompra.CPA_18_Eliminar_Prod_Pagar_CheckOut.CPA_18(CPA_18_Eliminar_Prod_Pagar_CheckOut.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- CPA_18 -->
</testsuite> <!-- belcorp.test.CarritoBolsaCompra.CPA_18_Eliminar_Prod_Pagar_CheckOut -->
